### DISCUSSION TITLE
VolumeJunction problem in THM

### DISCUSSION THREAD

**Discussion by SomnusYu** [idaholab/moose#27768] [Q&A Modules: Thermal Hydraulics]
Hi @joshuahansel and @GiudGiud ,

I'm doing the loop simulation in PWR using THM. Now, I just simulate a part of this PWR, including the downcomer, pipe, and low plenum, as shown in the picture:
![image](https://github.com/idaholab/moose/assets/118275988/93a34129-fa7b-4d19-a791-421b2d3b64a9)

But the solver did not converge. I tried a lot of ways to fix this problem, but nothing changed. I have no idea how to solve this problem. Could you please help me?

Here is my input file:
[core.txt](https://github.com/user-attachments/files/15533331/core.txt)

**Comment by GiudGiud** [comment]
Hello

I'm on travel right now and won't be able to look at this for another two weeks. 

Do we have access to your application ? I don't remember 

Did you follow the troubleshooting failed solves instructions on our website?
Guillaume

**Reply by SomnusYu** [reply]
Hi, I think I have solved this problem. Thanks!
